src/DateTimeInput/driver.js

```diff
@@ -0,0 +1,123 @@
+export default class DateTimeInputDriver
+{
+    constructor( wrapper )
+    {
+        this.wrapper     = wrapper;
+        this.cssMap      = wrapper.props().cssMap;
+        this.mainInput   = wrapper.find( 'input' ).at( 0 );
+        this.hourInput   = wrapper.find( 'input' ).at( 1 );
+        this.minuteInput = wrapper.find( 'input' ).at( 2 );
+        this.calendar    = wrapper.find( 'table button' );
+        this.prev        = wrapper.find( 'button' ).at( 1 );
+        this.next        = wrapper.find( 'button' ).at( 2 );
+    }
+
+    getMainInputValue()
+    {
+        return this.mainInput.node.value;
+    }
+
+    setMainInputValue( value )
+    {
+        const newValue = ( value == null ) ? '' : String( value );
+        const $input = this.mainInput;
```

Comment:
There’s no need for this intermediate variable: `$input`. (Also, in general, the `$` prefix shouldn’t be used – it’s not part of our code style).

Reply:
Fixed - just a leftover from somewhere else 😄